syntax = "proto3";

package security;

option java_multiple_files = true;
option java_package = "common.model.protos";
option java_outer_classname = "TenorTypeProtos";

enum TenorTypeProto {
    UNKNOWN_TENOR_TYPE=0;
    PERPETUAL=1;
    TERM=2;
}
